datafusion/physical-expr/src/expressions/cast.rs

```diff
@@ -680,7 +680,11 @@ mod tests {
         // Ensure a useful error happens at plan time if invalid casts are used
         let schema = Schema::new(vec![Field::new("a", DataType::Int32, false)]);
 
-        let result = cast(col("a", &schema).unwrap(), &schema, DataType::LargeBinary);
+        let result = cast(
```

Comment:
Is this change needed because arrow now supports  casting Int32 --> LargeBinary?